### ISSUE TITLE
FactionsDynmap Factionstyles edit?

### ISSUE THREAD

**Issue by Malformator** [MassiveCraft/Factions#1248]
Is there any chance of changing a currently existing factions colour scheme on the Dynmap?

**Comment by Malformator** [comment]
I was trying to edit the FactionStyles.json to add a new faction for a specific colour like so:


"WarZone": {
"lineColor": "#FF0000",
"fillColor": "#FF0000",
"boost": false
}
"boop": {
"lineColor": "#000000",
"fillColor": "#000000",
"boost": false
}
}
}

for some reason it's coming up as a problem for me, malformedjson..

Am I doing it wrong or is it not linked to the factions_faction folder but rather something more internal?

**Comment by oloflarsson** [comment]
Yes that should be supported in the config file already. Do you find it?
